m4/ax_pthread.m4

```diff
@@ -0,0 +1,522 @@
+# ===========================================================================
```

Comment:
I doubt it's worth adding this at all. Just use `$(CC) -pthread` and be done with it. Any older compiler isn't worth supporting, any older OS probably isn't in use any more.

Reply:
It was easier for me to pull the "official" test than write and debug a new one.